_posts/2017-11-15-code-review.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  "How To Do Code Review, So It Rocks!"
+date:   2017-11-15 12:00:00 -0400
+categories: culture code-review
+author: Janice Smith
+---
+
+ We do a lot of code review at Forward Financing. It's helped our team commit to delivering quality software and it has
+ turned out pretty great. It can be hard to know how to dive in, so I'm going to shed some light on how it can be approached.
+
+In general, the practice of code review helps peers collaborate on features. That collaboration brings fresh perspectives, exposing the coder and reviewer to new ways of approaching a problem they may not have thought of, while distributing that knowledge across the team. In order for this process to be successful, the review needs to be done well.
+
+Initially the coder should consider adding some context to a given Pull Request. I like to put a description of the changes and the feature, but it can also be nice to comment on specific lines that might be complex.  Also a link to the card or ticket can be a big help. When doing this, it is a good time for the coder to think about the PR as a whole.
+- Should this be more than one PR?
+- Should the code I write have any comments to clarify or document any of these changes?
+
+Once the PR is ready to be reviewed, it can be good to ask for the opinion of specific engineers while still opening it up to get more points of view.  It can be great to get the eyes of someone who has spent a lot of time in the codebase, but just keep in mind as the coder, you will get different kinds of feedback based on who is reviewing. I like to try to get a few different perspectives if I can.
```

Comment:
Once the PR is ready to be reviewed, asking for the opinion of specific engineers can be a good idea...

Reply:
~It can be great to~ Get the eyes of someone who has spent a lot of time in the codebase if possible,...